grakn/query/query_manager.py

```diff
@@ -17,13 +17,13 @@
 # under the License.
 #
 
-from typing import Callable, List
+from typing import Callable, List, Any
```

Comment:
are you using `Any`?

Reply:
Fixed in https://github.com/graknlabs/client-python/pull/148/commits/6406a2970bd49e9458fd226d93beafafdf37c449